media/pages/sk/podporte-nas/page.html

```diff
@@ -88,6 +88,18 @@ <h3>Výročné správy</h3>
           Výročná správa za rok 2016
         </a>
       </li>
+      <li>
+        <a href="{% page 'vyrocna-sprava-2018.pdf' %}">
```

Comment:
@martinmacko47 Co na to povies? 
![Screenshot 2021-06-13 at 17-52-07 Podporte nás Chcemvediet sk](https://user-images.githubusercontent.com/29918752/121814424-5dae3200-cc71-11eb-9eb2-0eae43753129.png)

Reply:
Do textu by som ikonky nedaval. Imho to ztazuje citanie.